README.md

```diff
@@ -105,13 +178,15 @@ prepResult codes (for images of same index in paper):
 - 1: images differed in size -- resized for comparison
 - 2: images differed in size -- not resized for comparison
 
+##### DEV
+For debugging purposes: When running the ERC-Checker with a NodeJS environment variable `DEV` set true, the result AND reject metadata will include an absolute path to the temp-directory used during the check.
 
 ### How to use the ERC-Checker module
 
 ```javascript
 const checker = require('<path>/<to>/erc-checker/index').ercChecker;  // import the ercChecker module, which is a function
 
-// head: checker(originalHTML, reproducedHTML, checkID, outputPath, saveDiffHTML, saveMetadataJSON, createParentDirectoriesInOutputPath, silenceDebuggers);    
+// head: checker(config);    
 ```
 
 The ercChecker function will return a Promise, which will be resolved on successful execution, or rejected on Error.
```

Comment:
There are at several ways of writing the name of this project... "ercChecker", "checker", "ERC-Checker". That is confusing.

Please always use simply "checker" in regular sentences, and use \` (backticks) to highlight inline code, for example here for `ercChecker` (the function name).